pkg/csi/cinder/nodeserver.go

```diff
@@ -448,13 +372,16 @@ func (ns *nodeServer) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandV
 	if len(volumePath) == 0 {
 		return nil, status.Error(codes.InvalidArgument, "Volume path not provided")
 	}
+	volCapability := req.GetVolumeCapability()
+	if volCapability != nil {
+		if volCapability.GetBlock() != nil {
+			return &csi.NodeExpandVolumeResponse{}, nil
+		}
+	}
```

Comment:
Could you please add a comment why do we need this and why do we return `&csi.NodeExpandVolumeResponse{}` once we detect that volume is a block device? Don't we need to expand a block device?

```suggestion

	volCapability := req.GetVolumeCapability()
	if volCapability != nil && volCapability.GetBlock() != nil {
		return &csi.NodeExpandVolumeResponse{}, nil
	}
```